content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -355,6 +355,12 @@ Used on: PersistentVolumeClaim
 
 This annotation will be added to dynamic provisioning required PVC.
 
+### volumes.kubernetes.io/controller-managed-attach-detach
+
+Used on: Node
+
+If a node has set the annotation volumes.kubernetes.io/controller-managed-attach-detach on itself, then its attach and detach operations are being managed by the controller. The controller will automatically inspect all nodes for this annotation and act according to whether it is present or not.
```

Comment:
I'd write:
```markdown
If a node has set the annotation `volumes.kubernetes.io/controller-managed-attach-detach`
on itself, then its storage attach and detach operations are being managed
by the FIXME
{{< glossary_tooltip text="controller" term_id="controller" >}}.
The FIXME controller automatically inspects all nodes for this annotation
and adapts its behavior according to whether the annotation is present or not.

The value of the annotation isn't important; if this annotation exists on a node,
then attach and detached are controller managed.
```

We should explain _which_ controller is doing that volume attach / detach management.

Reply:
@sftim  Could you please take a look here.

Reply:
OK to mark this as resolved.